test/TestOrchestrations.cs

```diff
@@ -256,5 +256,17 @@ public static string BigReturnValue([OrchestrationTrigger] DurableOrchestrationC
             int stringLength = ctx.GetInput<int>();
             return new string('*', stringLength);
         }
+

```

Comment:
I think the main thing is to fix the tests and the other points I made are minor. Thank you!